src/sentry/snuba/metrics/fields/snql.py

```diff
@@ -842,7 +848,9 @@ def on_demand_failure_count_snql_factory(
     )
 
 
-def on_demand_apdex_snql_factory(aggregate_filter, org_id, use_case_id, alias=None):
+def on_demand_apdex_snql_factory(
+    aggregate_filter: Function, org_id: int, use_case_id: UseCaseID, alias: Optional[str]
```

Comment:
Do we need to keep `None` as the default value for `alias`?

Reply:
What other default would you use?